relay/http.go

```diff
@@ -113,10 +116,15 @@ func (h *HTTP) Stop() error {
 func (h *HTTP) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	start := time.Now()
 
+	pingStatusResponse := DefaultHttpPingResponse
+	if (h.response != 0) {
+		pingStatusResponse = h.response
+	}
+
 	if r.URL.Path == "/ping" && (r.Method == "GET" || r.Method == "HEAD") {
-			w.Header().Add("X-InfluxDB-Version", "relay")
-			w.WriteHeader(http.StatusNoContent)
-			return
+		w.Header().Add("X-InfluxDB-Version", "relay")
```

Comment:
If this is going to return codes other than `204`, `Content-Length` should also be set to `0`.